pactsdk/add_liquidity.py

```diff
@@ -122,8 +133,20 @@ def build_effect(self) -> AddLiquidityEffect:
                     "Amount of minted liquidity tokens must be greater then 0.",
                 )
 
+        minimum_liquidity_tokens = max(
```

Comment:
I think this will be more readable if
```py
minimum_lt = round(minted_lt - (minted_lt * self.slippage_pct) / 100)
minimum_lt = max(0, minimum_lt)
```